src/hotspot/cpu/x86/x86_64.ad

```diff
@@ -7612,18 +7746,6 @@ instruct addP_rReg_imm(rRegP dst, immL32 src, rFlagsReg cr)
 
 // XXX addP mem ops ????
```

Comment:
This comment line is for leaP_rReg_imm, which should also be removed?

Reply:
I believe it is for the missing rules of `AddP` with memory operands such as `addP_mem_rReg`